index.js

```diff
@@ -21,7 +21,12 @@ function resetReducer(initialState, reducer){
   };
 }
 
-function promiseHandler(type, namespace){
+function promiseHandler(type, namespace, options){
```

Comment:
was thinking something more like `function promiseHandler(type, options){`

and 
```javascript
 if(typeof namespace === 'string') {
  options = {namespace: namespace};
}
```
and finally:

```javascript
var name = (options.namespace ? options.namespace + '-' : '') + type;
```